core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/clusterproxy/ClusterProxyService.java

```diff
@@ -111,17 +115,22 @@ public void deregisterCluster(Stack stack) {
     private ConfigRegistrationRequest createProxyConfigRequest(Stack stack) {
         ConfigRegistrationRequestBuilder requestBuilder = new ConfigRegistrationRequestBuilder(stack.getResourceCrn())
                 .withAliases(singletonList(clusterId(stack.getCluster()))).withServices(serviceConfigs(stack));
-        if (stack.getTunnel().useCcm()) {
+        if (stack.getTunnel().useCcmV1()) {
             requestBuilder.withAccountId(getAccountId(stack)).withTunnelEntries(tunnelEntries(stack));
+        } else if (stack.getTunnel().useCcmV2()) {
+            requestBuilder.withAccountId(getAccountId(stack)).withCcmV2Entries(ccmV2Configs(stack));
```

Comment:
for reregistration you add knox url to the request. Is it missing here on purpose?

Reply:
I have not made any changes in this flow, first createProxyConfigRequest is sent. then knoxUrl is registered with registerGateway call. But for createProxyConfigReRegisterRequest knoxUrl was always passed.

Reply:
ok, I see that now. thanks for clarification